fix neume image path -clean

yinanazhou opened the pull request:
Corrected neumes image path, and uploaded missing images in the `neume `folder (cleaned). [Notice](https://github.com/music-encoding/music-encoding/discussions/829): the images will only show up when using **command lines** to build HTML files. To make the **Oxygen** method work, the `image `folder needs to be pasted to `music-encoding\source\web` manually.

Commit:
fix neume image -clean

ahankinson commented on source/docs/06-neumes.xml:
```diff
@@ -125,9 +125,17 @@
+                          </figure>
                         </cell>
-                        <cell>!<ref target="/guidelines/images/v4/modules/neumes/NEUME-non-con.png">Non-connected</ref>
+                        <cell>
```
This change seems to lose the 'Connected' and 'Non-connected' labels?

ahankinson commented on source/docs/06-neumes.xml:
```diff
@@ -996,7 +1143,11 @@
-                        <cell>!<ref target="/guidelines/images/v4/modules/neumes/quilisma.png">Quilisma</ref>
+                        <cell>
+                          <figure>
+                            <head/>
```
Is the empty `head` tag required?

yinanazhou commented:
Thanks for pointing that out. The labels can be put inside the `<head>` message. The resulting figure caption is going to be `Figure XX. Connected` if we add the label in the empty `<head>` message. We can add the labels if necessary.

ahankinson commented:
They seem necessary?

yinanazhou commented:
OK! We will add them right now!

yinanazhou committed:
Update head message

Add label to head message for images in neume doc according to the music-encoding repo

ahankinson commented:
Great! Thank you.

Just to make it easier to review without needing to download and run the code locally, could you please provide a screenshot of what it looks like now? Just one or two examples showing how the label and image look together would be sufficient.

yinanazhou commented:
Hi, we just built it using command lines, and it looks like this now:
![connected](https://user-images.githubusercontent.com/74484576/123129656-1ca8e100-d41a-11eb-8d76-d50bd3a88127.png)
![nc](https://user-images.githubusercontent.com/74484576/123129660-1d417780-d41a-11eb-89a0-e780fa62ea50.png)

ahankinson commented:
OK -- do you know where those funny `:-------:` things come from?

ahankinson commented:
Ah, looking at it more, it seems that the 'connected' and 'non-connected' labels are in some sort of table? And the `:-----:` is maybe a hangover from the markdown conversion? @bwbohl @kepper do you know anything about this?

Should we clean those up now, or should we open another issue for it?

yinanazhou commented:
They are in the [original version](https://github.com/music-encoding/music-encoding/blob/develop/source/docs/06-neumes.xml). We did not change it.
Example (line 123-126): 
```
<row>
      <cell>:---------:</cell>
      <cell>:---------:</cell>
</row>
```

kepper commented:
Yes, if I remember correctly, those are artifacts from how figures and captions were organized "creatively" in Markdown. Would be perfect to clean that up while you're at it now… (pretty please ;-) From my perspective, those examples don't need to be written as table, and using a table for layout purposes is generally questionable, isn't it?

ahankinson commented:
@yinanazhou would you be able to make the changes?

musicEnfanthen commented:
The hyphens within colons mark a "center-aligned" table cell in markdown: https://docs.github.com/en/github/writing-on-github/working-with-advanced-formatting/organizing-information-with-tables#formatting-content-within-your-table
All those [`<rows>` artifacts](https://github.com/music-encoding/music-encoding/pull/832#issuecomment-866966889) could be safely removed.

yinanazhou commented:
@ahankinson yes, sure. We will do it now.

carrieeex committed:
Merge branch 'music-encoding:develop' into fix-neume-img-new

carrieeex committed:
delete the <row> artifacts

delete the artifacts in <row> as suggested in PR #832 (https://github.com/music-encoding/music-encoding/pull/832)

yinanazhou commented:
Hi @ahankinson and @martha-thomae, this is how it looks like now:
![image](https://user-images.githubusercontent.com/74484576/123429893-7e418a80-d595-11eb-989b-e36f3b49c8b8.png)

ahankinson commented:
Looks much better! Thank you.

@martha-thomae @kepper would you be willing to be the second approver on this PR? If so, feel free to merge if you're happy.
